src/controls/listView/ListView.tsx

```diff
@@ -386,7 +386,14 @@ export class ListView extends React.Component<IListViewProps, IListViewState> {
     // Check if the URL property is specified
     if (field.linkPropertyName) {
       return (item: any, index?: number, column?: IColumn) => { // eslint-disable-line @typescript-eslint/no-explicit-any
-        return <a href={item[field.linkPropertyName]}>{item[column.fieldName]}</a>;
+        
+        // Testing to see if Url value is valid... has a value, is a string, and either starts with http or /sites/
+        const isValid = item[field.linkPropertyName] && typeof item[field.linkPropertyName] === "string" &&
+            ( item[field.linkPropertyName].indexOf("/sites/") === 0 || item[field.linkPropertyName].indexOf("http") === 0 ) ? true : false;
```

Comment:
@mikezimm sorry for the delay on this.
The /sites/ validation will fail for example if the site is for example the root site of the tenant or is under /teams/.
Maybe check if the hyperlink string containst the site relative url? 
As for http, I think you can go even further with "https://", what do you think?

Reply:
@joelfmrodrigues , One other note I forgot to mention:

For the validation check of /sites/:

I am checking to see if the url starts with /sites/ because we can pass in text columns for the url.
In those cases, we sometimes just start the url with /sites/ to keep it shorter.
But if it's the full Url it will always start with http....

If it's a normal Url column, column validation forces proper validation in the first place.

Hope that explains the logic.

Reply:
> Hi @mikezimm , just checking if you need any help with the PR?

I think I just manually edited the file.  Did I do it wrong or did I miss  a step?